recipe/meta.yaml

```diff
@@ -7,8 +7,8 @@ package:
   version: {{ version }}
 
 source:
-  fn: graphviz-{{ version }}.zip
-  url: https://files.pythonhosted.org/packages/source/g/graphviz/graphviz-{{ version }}.zip
```

Comment:
This is now actually the preferred URL, see https://bitbucket.org/pypa/pypi/issues/438/backwards-compatible-un-hashed-package#comment-27606213